tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Install required packages
+  apt:
+    name: "{{ item }}"
+  with_items:
+    - python
+    - python-flask
+    - python-requests
+    - python-pillow
+
+- name: Clone reposity
+  git:
+    repo: 'https://github.com/softdevteam/mattermost-github-integration.git'
+    dest: "{{ mattermost_github_integration_install_path }}"
+    depth: 1
```

Comment:
Is `update: yes` the default? Also, please link to the original repo in the README

Reply:
its default and fixed